Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift

```diff
@@ -85,43 +60,31 @@ final class PledgePaymentMethodsViewController: UIViewController {
     _ = self.view
       |> checkoutBackgroundStyle
 
-    _ = self.cardsStackView
-      |> cardsStackViewStyle
-
-    _ = self.applePayButton
```

Comment:
Removal of views in this PR to reduce conflicts with the work in #1151 and #1154.